CustomLookupTest/test.cs

```diff
@@ -0,0 +1,286 @@
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using Tests;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Linq.Expressions;
+using System.Reflection.Metadata;
+using System.Threading;
+using System.Timers;
+using System.Linq;
+using System.Runtime.ExceptionServices;
+/// <summary>
+/// This is my attempt at a unit test. I took information from VSE documentation and the team's current SkillsetsTest.cs
+/// </summary>
+namespace Test
```

Comment:
>Test [](start = 10, length = 4)

name this file something more descriptive? eg: LookupTests for pattern matching unit tests or WebResponseTests for integration tests validating response codes